lib/app.js

```diff
@@ -485,6 +483,24 @@ var makeApp = function(configBase, callback) {
 
         app.use(xssCheck.xssCheck);
 
+        if (config.noweb) {
+            // Not serve content of 'javascript' or 'template' folder in noweb,
+            // this before serve publics for prevent response
+            // and after other middlewares to let them work well
+            app.use(function(req, res, next) {
+                var path = req.originalUrl.split("/")[1];
+
+                if (path.match(/javascript|template/)) {
+                    next(new HTTPError("Not Found", 404));
+                } else {
+                    next();
+                }
+            });
+        }
+
+        // Serve publics
```

Comment:
Can probably get rid of this comment - it's pretty obvious what this middleware does IMO

Reply:
sure